tooling/monitoring-dashboard/README.md

```diff
@@ -0,0 +1,34 @@
+# Monitoring Dashboard for WSO2 Micro Integrator
+
+## Building from the source
+
+- ### Setting up the development environment
+1. Install Node.js [8.X.X](https://nodejs.org/en/download/releases/)
+2. Fork the [repository](https://github.com/wso2/micro-integrator)
+3. Clone your fork into any directory
+4. `cd` into cloned directory and then cd into `micro-integrator/tooling/monitoring-dashboard`.
+5. Perform a maven build ```mvn clean install```
+6. wso2mi-monitoring-dashboard-version.zip can be found in
+ `micro-integrator/tooling/monitoring-dashboard/distribution/target`
+ 
+- ### Running
+- Extract the compressed archive generated to a desired location.
+    ```
+    cd to the DASHBOARD-HOME/bin
+    
+    Execute `dashboard.sh or dashboard.bat as appropriate.
+    ```
+
+
+- Load the login page with the dashboard context. i.e: https://localhost:9743/dashboad
+- Please note that in order to connect the monitoring dashboard to a micro integrator instance, the
+  said instance should have the management api enabled. This can be done by starting the MI with 
```

Comment:
management api -> Management API 
and this needs to have a link to the documentation
the MI -> WSO2 MI